eks/mng/mng.go

```diff
@@ -167,3 +169,18 @@ func (ts *tester) Delete() error {
 	ts.cfg.EKSConfig.AddOnManagedNodeGroups.Created = false
 	return ts.cfg.EKSConfig.Sync()
 }
+
+func (ts *tester) scaleMNG() (err error) {
```

Comment:
Can we move this as a separate package e.g. https://github.com/aws/aws-k8s-tester/tree/master/eks/mng/version-upgrade